src/sass/style.scss

```diff
@@ -0,0 +1,2 @@
+@import "src/components/home-page/home-page";
```

Comment:
what is the purpose of this file?

Reply:
This file defines a style for the home page